config/locales/fil/contacts.fil.yml

```diff
@@ -0,0 +1,11 @@
+fil:
+  contacts:
+    new:
+      request-edit-for-restroom: 'Humiling ng edit para sa %{restroom_name}'
```

Comment:
Still slightly related: Is there a better translation for this that doesn't use "edit" as a "loan word"? Or should it stay the way it already is?

Reply:
I've change this since this has a tagalog translation. I overlooked this when I checked before pushing the last commit. Thanks for noticing!